app/models/mentor/discussion.rb

```diff
@@ -33,7 +33,9 @@ class Mentor::Discussion < ApplicationRecord
   scope :in_progress_for_student, -> { where(status: %i[awaiting_student awaiting_mentor mentor_finished]) }
   scope :finished_for_student, -> { where(status: :finished) }
   scope :finished_for_mentor, -> { where(status: %i[mentor_finished finished]) }
-  scope :not_negatively_rated, -> { where(rating: [nil, 3, 4, 5]) }
+  scope :not_negatively_rated, -> { where(rating: [nil, :acceptable, :good, :great]) }
+  scope :satisfactory_rated, -> { where(rating: %i[acceptable good great]) }
```

Comment:
Is `satisfactory_rated` the right word? It feels awkward, but I don't know what a better word would be.

Reply:
This scope is only used in the recalculating of the satisfaction rating. Want me to inline it? I didn't yet because I could imagine it being useful somewhere else later on.

Reply:
1.`satisfactorily` is probably the word you're after. But I'd probably have `rated_acceptable_or_better` or something.
2. Yeah - I think inline it.